packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -128,16 +128,6 @@ html.interface-interface-skeleton__html-container {
 	border-bottom: $border-width solid $gray-200;
 	z-index: z-index(".interface-interface-skeleton__header");
 	color: $gray-900;
-
-	// On Mobile the header is sticky.
-	position: sticky;
-	top: 0;
-
-	// Cancel the fixed positioning used on Mobile.
-	@include break-small() {
-		position: initial;
-		top: 0;
-	}
```

Comment:
With the interface skeleton components (header, body, and footer) set to display in a flex column, and overflow scrolling limited to the skeleton body, I can't see a reason we need the header to be declared as sticky--it effectively already is sticky from the flex layout.

(`position: initial` was causing the block toolbar to overlap the navigation panel at small - medium screen sizes. An alternative to removing these styles would be setting `position: relative` here, instead.)

Reply:
There was a good reason for it, notably that some of the mobile safari overscroll bounce got locked when using flex for layouts rather than letting the HTML element scroll directly.

In testing with the simulator just now, it does appear to have been solved, so it's _probably_ fine to move forward with this one, but I'd encourage just a quick sanity check if you're able to test on a physical device, or the simulator that comes with Xcode. Ping me if you need some more context!

Reply:
Thanks! I just spent some time testing this on an iPhone (actual device) with Safari and I couldn't see a difference in scrolling behavior between this and trunk.

(I did see some weird things that happen occasionally in the site editor with a lot of quick scrolling up and down and selecting different blocks, mostly related to how mobile Safari handles input focusing and opening the on screen keyboard... for example, after tapping a block with text input, being immediately scrolled to the bottom of the page, or losing the ability to scroll down until a few seconds have passed after tapping. But the same behavior is present on trunk so I don't think it has to do with the sticky header css removed by this PR.)